docs/sql/system.txt

```diff
@@ -677,6 +677,85 @@ deactivate the stats tracking. This will also wipe all existing log entries::
     reset the value of ``stats.enabled`` to its default which is false.
 
 
+Settings checks
+===============
+
+The table ``sys.checks`` contains misconfigured settings of a cluster.
+
+The table schema is following:
+
++------------------+----------------------------------+-------------+
+|       Name       |           Description            | Return Type |
++==================+==================================+=============+
+| id               | The unique check id.             | Integer     |
++------------------+----------------------------------+-------------+
+| severity         | The level of severity.           | Integer     |
+|                  | 1 - INFO, 2 - WARN, 3 - ERROR    |             |
++------------------+----------------------------------+-------------+
+| description      | The description message for the  | String      |
+|                  | setting check.                   |             |
++------------------+----------------------------------+-------------+
+| passed           | The flag determines whether the  | Boolean     |
+|                  | check for the setting has passed.|             |
++------------------+----------------------------------+-------------+
+
+Currently, the following cluster settings are supported::
+- ``discovery.zen.minimum_master_nodes``
+- ``gateway.recover_after_nodes``
+- ``gateway.recover_after_time``
+- ``gateway.expected_nodes``
+
+Settings checks::
+
+All of the settings must be present in the cluster configuration.
+
+The check for the ``discovery.zen.minimum_master_nodes`` settings verifies that
+the minimum number of nodes is greater than the half of maximum number of
+nodes in the cluster. The checks ``id`` is ``1``.
+
+  (N / 2) + 1 < M
+
+where ``N`` is the number of nodes in the cluster, ``M`` is the number of
+minimum master nodes.
+
+The check for the ``gateway.expected_nodes`` setting checks that the number of
+nodes which should be waited for the immediate cluster recovery must be equal
+to the maximum number of nodes in the cluster. The check ``id`` is ``2``.
+
+The check for the ``gateway.recover_after_nodes`` verifies that the number of
+started nodes before the cluster starts recovery must be between half of nodes
+in the cluster and the number of expected nodes. The check ``id`` is ``3``.
+
+  (N / 2) + 1 <= R <= E
+
+where ``N`` is the number of nodes in the cluster, ``R`` is the number of
+recovery nodes, ``E`` is the number of expected nodes.
+
+The check for the ``gateway.recover_after_time`` settings verifies that the
+time before recovery starts must not be less than default value. The check
+``id`` is 4.
+
+For instance, if the ``gateway.expected_nodes`` and
+``discovery.zen.minimum_master_nodes`` settings were misconfigured then the
+SQL query bellow would have the following results::
+
+  cr> select *
```

Comment:
`cr>` blocks are executed when running doc tests
this means, that this example will probably always fail